iib/workers/tasks/fbc_utils.py

```diff
@@ -69,3 +70,23 @@ def get_hidden_index_database(from_index: str, base_dir: str) -> str:
     os.makedirs(os.path.dirname(base_db_file), exist_ok=True)
     _copy_files_from_image(from_index, conf['hidden_index_db_path'], base_db_file)
     return base_db_file
+
+
+def merge_catalogs_dirs(src_config: str, dest_config: str):
+    """
+    Merge two catalog directories by replacing everything from src_config over dest_config.
+
+    :param str src_config: source config directory
+    :param str dest_config: destination config directory
+    :return: path to the dest_config
+    :rtype: str
+    """
+    for conf_dir in (src_config, dest_config):
+        if not os.path.isdir(conf_dir):
+            msg = f"config directory does not exist: {conf_dir}"
+            log.error(msg)
+            raise IIBError(msg)
+
+    log.info("Merging config folders: %s to %s", src_config, dest_config)
+    shutil.copytree(src_config, dest_config)
```

Comment:
are we sure we can use `shutil.copytree` as is? 

From the docs, it looks like this only works if the destination doesn't exist (https://docs.python.org/3/library/shutil.html#shutil.copytree). IIUC, In our case, we want to overwrite the contents of the destination so we might have to specify `dirs_exist_ok =True` since the default is false.

Reply:
You are right, thank you.   Added `dirs_exist_ok`